sklearn/utils/tests/test_linear_assignment.py

```diff
@@ -1,12 +1,17 @@
 # Author: Brian M. Clapper, G Varoquaux
 # License: BSD
 
+# TODO: Remove this test module as the methods being tested
+# have been replaced by SciPy methods
+
 import numpy as np
 
 # XXX we should be testing the public API here
 from sklearn.utils.linear_assignment_ import _hungarian
```

Comment:
you need to ignore the warning here too.. or just put the import inside the test function

Reply:
moved the import inside test function